components/twitter/README.md

```diff
@@ -15,4 +15,17 @@ In order to connect your Twitter account to Pipedream, you'll need to create a d
 4. Enter this URI in the "Callback URI / Redirect URI": `https://api.pipedream.com/connect/oauth/oa_gk6iBa/callback`
 5. Enter a website URL
 6. Click "Done" (the Client ID and Client Secret that are shown on the next page are not required for this setup)
-7. Enter the API Key and API Key Secret when prompted in Pipedream (if you lose them and need to regenerate new keys, you can do so from the "Keys and tokens" tab in your app's settings in Twitter's developer dashboard, in the "Consume Keys" section)
\ No newline at end of file
+7. Enter the API Key and API Key Secret when prompted in Pipedream (if you lose them and need to regenerate new keys, you can do so from the "Keys and tokens" tab in your app's settings in Twitter's developer dashboard, in the "Consumer Keys" section)
+
+# Troubleshooting
+Below are some of the most common issues we see.
+
+## Issues Connecting Twitter
+If you run into an error during the app connection process, make sure to doublecheck the below configurations:
+1. Ensure that you've entered the correct `developer_consumer_key` and `developer_consumer_secret` in Pipedream. These map to Twitter's "API Key" and "API Key Secret" in the "Keys & Tokens" section in their dashboard.
+2. Make sure the "Callback URI / Redirect URL" that you configure within the "User Authentication settings" for your app in Twitter's developer console exactly matches: `https://api.pipedream.com/connect/oauth/oa_gk6iBa/callback`
```

Comment:
```suggestion
1. Ensure that you've entered the correct `developer_consumer_key` and `developer_consumer_secret` in Pipedream. These map to Twitter's "API key" and "API key secret" in the "Keys & Tokens" section in their dashboard.
```

Reply:
Changed "API Key" and "API Key Secret" to "API key" and "API key secret" to follow Google's Developer docs style guide (https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings)